src/assets/fonts/fonts.scss

```diff
@@ -0,0 +1,13 @@
+@font-face {
+  font-family: 'SSTW20-Bold', sans-serif;
```

Comment:
u can't use sans-serif here. It will work only for: font-family: 'SSTW20-Bold' 
check settings in your editor.
https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-family